.travis.yml

```diff
@@ -1,12 +1,32 @@
 language: ruby
+cache: bundler
 sudo: false
 rvm:
-  - 2.1
-  - 2.2
+  - 2.1.10
+  - 2.2.9
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
```

Comment:
Is there a reason for specifying patch levels?

Reply:
It seems to vary over time - it's not totally stable over time and leads to non-obvious build failures when the problem comes back after a while.  I've seen several projects go to this method to avoid that, Rails being an obvious example.

When I did the first commit of this patch, 2 or 3 of them were out of date so I made this change and force-pushed over the top.